edgelet/hosting/git_push.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
What is this file for?

Reply:
It came with the generated swagger client, removed it